docs/about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Ibhicheckesho | About</title>
+  </head>
+  <body>
+    <header id="header-about">
+      <div class="container">
+        <nav id="main-nav">
+          <div class="container">
```

Comment:
Super cool navbar layout. Well done :+1: 
I think you missed the social media navigation bar at the top. :grinning: 
Let's add it :point_down: 

 ![](https://camo.githubusercontent.com/96bd23868b8867313e32b0dd0bd61c02273f1ccfff54263caef020c9b2a779c2/68747470733a2f2f692e696d6775722e636f6d2f6e733846646b6b2e706e67)